src/gossip_agent/message_typing.rs

```diff
@@ -159,7 +160,7 @@ impl GraphcastMessage {
             sig.to_string(),
         );
 
-        println!("{}{:#?}", "Encode message: ".cyan(), message,);
+        info!("{}{:#?}", "Encoded message: ".cyan(), message,);
```

Comment:
`Message` can be long, so maybe make it `debug` level?

Reply:
✅